PWGHF/DataModel/HFSecondaryVertex.h

```diff
@@ -803,7 +803,64 @@ auto InvMassXToJpsiPiPi(const T& candidate)
 {
   return candidate.m(array{RecoDecay::getMassPDG(443), RecoDecay::getMassPDG(kPiPlus), RecoDecay::getMassPDG(kPiPlus)});
 }
+template <typename T>
+auto QX(const T& candidate)
+{
+  array<float, 3> piVec0;
+  array<float, 3> piVec1;
+
+  piVec0[0] = candidate.pxProng1();
+  piVec0[1] = candidate.pyProng1();
+  piVec0[2] = candidate.pzProng1();
+
+  piVec1[0] = candidate.pxProng2();
+  piVec1[1] = candidate.pyProng2();
+  piVec1[2] = candidate.pzProng2();
+
+  double massPi = RecoDecay::getMassPDG(kPiPlus);
+
+  auto arrayMomenta = array{piVec0, piVec1};
+  double massPiPi = RecoDecay::M(arrayMomenta, array{massPi, massPi});
+
+  // PDG mass, as reported in CMS paper https://arxiv.org/pdf/1302.3968.pdf
+  double massMuMu = RecoDecay::getMassPDG(o2::analysis::pdg::kJpsi);
```

Comment:
This is just a J/psi mass. Why is it called `massMuMu`?

Reply:
Name changed as massMuMu -> massJpsi